src/disco/topo/fd_topo.h

```diff
@@ -142,6 +142,7 @@ typedef struct {
       ushort shred_listen_port;
       ushort quic_transaction_listen_port;
       ushort legacy_transaction_listen_port;
+      ushort repair_listen_port;
```

Comment:
Dont you need both serve and listen? Or this should clarify which one

Reply:
For the purposes of filtering via the net tile we only need listen in this ctx. I removed the net tile change i made from this PR which is why this feels unclear.

Reply:
I meant to say serve or intake? It's unclear from "listen" which one and I still don't know..